include/flatbuffers/flatbuffers.h

```diff
@@ -1793,9 +1802,34 @@ class FlatBufferBuilder {
   /// where the vector is stored.
   Offset<Vector<Offset<String>>> CreateVectorOfStrings(
       const std::vector<std::string> &v) {
-    std::vector<Offset<String>> offsets(v.size());
-    for (size_t i = 0; i < v.size(); i++) offsets[i] = CreateString(v[i]);
-    return CreateVector(offsets);
+    return CreateVectorOfStrings(v.begin(), v.end());
```

Comment:
`cbegin() + cend()` will be more safe.

Reply:
I switched to them, but because the input was a `const vector` i think `begin/end` are already const qualified.